slides/src/content/hello.md

```diff
@@ -0,0 +1,158 @@
+<!-- .slide: data-background="../content/images/title-slide.jpg" -->
+
+## Building Applications with Angular
+
+# Hello, World
+
+---
+
+## What's Where
+
+```
+├── src
+│   ├── index.html
+│   ├── main.ts
+│   ├── app/…
+```
+
+The root directory has:
+
+- `index.html`: contains the entire application
+  - Everything interesting right now is one level down in `app`
+- `main.ts`: bootstraps the application
+  - This can change from platform to platform
+  - So that everything else doesn't have to
+
+---
+
+## What's Where
+
+```
+├── src
+│   ├── app
+│   │   ├── app.component.css
+│   │   ├── app.component.html
+│   │   ├── app.component.spec.ts
+│   │   ├── app.component.ts
+│   │   ├── app.module.ts
+```
+
+The *application* as a whole has:
+
+- `app.module.ts`: what to load and how to launch
+- Main application:
+  - Code: `app.component.ts`
+  - HTML: `app.component.html`
+  - CSS: `app.component.css`
+  - Test spec: `app.component.spec.ts`
+
+---
+
+## What's in the Component?
+
+_src/app/app.component.ts_
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'app works!';
+}
+```
+
+- `import` loads definition of Angular components
+
+---
+
+## What's in the Component?
+
+_src/app/app.component.ts_
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'app works!';
+}
+```
+
+- `@Component` is a *decorator* that adds metadata to the `AppComponent` class
+  - `selector` tells Angular to use this class to fill in uses of `<app-root></app-root>`
+  - `templateUrl` tells it where to find the HTML to use for filling in
+  - `styleUrls` is a list of CSS style files to apply to this component
+
+---
+
+## What's in the Component?
+
+_src/app/app.component.ts_
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'app works!';
+}
+```
+
+- The `AppComponent` class defines how this component behaves
+- So far, it just has a single instance variable and no actions
+- Note that we don't need to declare the type of `title`
+  - TypeScript does *type inference* to figure that out
+
+---
+
+## Notes on Components
+
+- Components are the core building blocks of Angular applications
+  - Application logic + display
+  - Everything on a page should be associated with some component
+- Can put HTML and CSS inline by using `template` and `style` as keys
+
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: '<h1>{{title}}</h1>,          // inline HTML
```

Comment:
You need to close the template parameter with a single quote. Syntax-highlighting gets confused.